pkgs/applications/audio/zrythm/default.nix

```diff
@@ -0,0 +1,115 @@
+{ stdenv
+, fetchFromGitHub
+, bash
+, meson
+, ninja
+, cmake
+, pkgconfig
+, python3Packages
+, help2man
+, python3
+, libxml2
+, audec
+, gtk3
+, wrapGAppsHook
+, guile
+, gtksourceview
+, lilv
+, libcyaml
+, libyaml
+, fftw
+, fftwFloat
+, libjack2
+, ffmpeg
+, carla
+, rtaudio
+, rtmidi
+, SDL2
+, libgtop
+, xdg_utils
+, pandoc
+, texi2html
+, alsaLib
+, git
+, libsndfile
+, libsamplerate
+, sratom
+, serd
+, sord
+, chromaprint
+, rubberband
+, pcre
+, bash-completion
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zrythm";
+  version = "0.8.252";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0m7i8f5y3lvkbflys0iblkl6ksw6kn2b64k6gy10yd549n8adx69";
+  };
+
+  prePatch = ''
+    # doesn't work:
+    # patchShebangs resources/gen_gtk_resources_xml_wrap.sh
+    substituteInPlace resources/gen_gtk_resources_xml_wrap.sh --replace "/usr/bin/env sh" "${bash}/bin/sh"
+  '';
+
+  nativeBuildInputs = [ audec gtk3 wrapGAppsHook meson ninja cmake pkgconfig python3Packages.sphinx help2man python3 libxml2 ];
```

Comment:
Could you make this into multiple lines as well?

Reply:
In new expressions, `pkg-config` should be used instead of the `pkgconfig` alias.

Reply:
Also could you add a comment to libraries, why they are needed during build time? For example `gtk3 # needed for gtk-update-icon-cache`.

Reply:
Are you sure `cmake` is needed? Meson supports it for finding dependencies but typically, `pkg-config` will be sufficient.

Reply:
All done, except the comments.
I don't remember all the messages, so I'd have to comment out all of the dependencies, and then 27 times do: build, copy/edit the error,  and enable one of dependencies.
I'm not sure if this niche package is worth that, sorry.